src/distPlus/distribution/Continuous.re

```diff
@@ -3,30 +3,45 @@ open Distributions;
 type t = DistTypes.continuousShape;
 let getShape = (t: t) => t.xyShape;
 let interpolation = (t: t) => t.interpolation;
-let make = (interpolation, xyShape, knownIntegralSum): t => {
+let make = (interpolation, xyShape, integralSumCache, integralCache): t => {
```

Comment:
We should have done this sooner, but we really should ~name the arguments, especially as they hit 4. (Also, for the discrete distribution)

Reply:
I could also see something like having defaults, with xyShape at the end unnamed. Like,
```let make = (~interpolation=`Linear, ~integralSumCache=None, ~integralCache=None, xyShape): t => {``